packages/commonwealth/server.ts

```diff
@@ -91,12 +90,24 @@ const start = async () => {
 
       // checking the DYNO env var ensures this only runs on one dyno
       if (config.APP_ENV === 'production' && process.env.DYNO === 'web.1') {
+        const { dispatchSDKPublishWorkflow } = await import(
+          './server/util/dispatchSDKPublishWorkflow'
+        );
         dispatchSDKPublishWorkflow().catch((e) =>
           log.error(
             `Failed to dispatch publishing workflow ${JSON.stringify(e)}`,
           ),
         );
       }
+
+      // bootstrap bindings when in dev mode
+      if (config.NODE_ENV === 'development') {
+        const { bootstrapBindings, bootstrapRelayer } = await import(
+          './server/bindings/bootstrap'
+        );
+        await bootstrapBindings();
+        await bootstrapRelayer();
+      }
```

Comment:
Can you add an env var to bypass this? I prefer to run the consumer separately for logging purposes